src/components/Sidebar/index.jsx

```diff
@@ -75,11 +80,39 @@ const Sidebar = ({
     <MessageForm courseId={courseId} shouldAutofocus unitId={unitId} />
   );
 
+  const getDaysRemainingMessage = () => {
+    if (auditTrialDaysRemaining > 1) {
+      const irtl = new Intl.RelativeTimeFormat({ style: 'long' });
```

Comment:
Nit: Is this meant to be:
```suggestion
      const intl = new Intl.RelativeTimeFormat({ style: 'long' });
```

Reply:
Oh wait, nvm, is this the relative time format? Could make this a little more descriptive, but totally your call.